lib/src/bridge/scanning_mixin.dart

```diff
@@ -1,19 +1,29 @@
 part of _internal;
 
 mixin ScanningMixin on FlutterBLE {
-  Stream<ScanResult>? _scanEvents;
-
-  Stream<ScanResult> _prepareScanEventsStream() {
-    return const EventChannel(ChannelName.scanningEvents)
-        .receiveBroadcastStream()
-        .handleError(
-          (errorJson) => throw BleError.fromJson(jsonDecode(errorJson.details)),
+  Stream<ScanResult>? __scanEvents;
```

Comment:
How about `_activeScanEvents` or `_currentScanStream`? I don't like having a variable discerned only by having one more dash in front - too easy to fool the eye, especially with IDE suggestion.

Reply:
the goal was to make it super-duper private, and to denote that it wasn't meant to be used from the class scope only from the scope of the actual interface being:
`get _scanEvents` getter and `_resetScanEvents()` to indirectly interface that state variable in the class.
`_activeScanEvents` works too ofc. ;)